Automattic/wp-calypso
# Add Jetpack Cloud Site Selection

#### Changes proposed in this Pull Request

* Add Site selection when navigating to `/scans` or `/backup` without a site slug

<img width="1249" alt="Screen Shot 2020-02-24 at 1 54 44 PM" src="https://user-images.githubusercontent.com/2810519/75194986-f97afe00-570d-11ea-85e2-fb3a92fea843.png">

#### Next Steps

* Current Site Indicator
* Hide Sidebar on site selector
* Tweak sidebar appearance
* Update sidebar url schemes
* Handle no sites/no eligible sites/site identifier no found and site identifier not eligible
* Update additional routes with site selection

#### Testing instructions

##### Test with one non-atomic Jetpack site

1. Go directly to `/backups` on Jetpack Cloud
2. Confirm the that the site slug of your one Jetpack site is automatically added to the URL and that the correct siteId is displayed on the page
3. Repeat steps 1-2 for `/scan`

##### Test with two or more non-atomic Jetpack sites

1. Go directly to `/backups` on Jetpack Cloud
2. Confirm that a site selector matches the screenshot above
3. Chose a site
4. Confirm the that the site slug of your one Jetpack site is added to the URL and that the correct siteId is displayed on the page
5. Repeat steps 1-4 for other sites and `/scan`
